protostar/starknet_gateway/block_explorer/stark_scan_block_explorer_test.py

```diff
@@ -0,0 +1,61 @@
+import pytest
```

Comment:
I think this test does not bring any value. Tested functions are trivial by definition? Are we sure we want this?

Reply:
Removed tests